src/components/form/form.js

```diff
@@ -601,11 +671,19 @@ class Form extends React.Component {
    */
   formFooter = () => {
     const save = this.props.showSummary ? this.saveButtonWithSummary() : this.saveButton();
+    let padding = this.props.stickyFooterPadding;
+
+    if (padding && !padding.match(/px$/)) {
+      padding = `${padding}px`;
```

Comment:
We usually define a group of preset values. Do you think we should match this here?